ecole-pratique-des-hautes-etudes-sciences-historiques-et-philologiques.csl

```diff
@@ -0,0 +1,620 @@
+        <group delimiter=" " font-style="normal" prefix=", ">
+          <group delimiter=" " suffix=".">
+            <label variable="page" form="short"/>
+            <text variable="page" form="short"/>
+          </group>
+        </group>
+      </else-if>
+      <else-if type="article-newspaper speech paper-conference" match="any">
+        <group font-style="normal">
+          <group delimiter=" " prefix=", ">
+            <label variable="page" form="short"/>
+            <text variable="page"/>
+          </group>
+        </group>
+      </else-if>
+    </choose> 
+  </macro>
+  <macro name="ISBN">
+    <text variable="ISBN" prefix=". ISBN : " suffix="."/>
+  </macro>
+  <macro name="archive">
+    <group suffix="." prefix=". ">
+      <group delimiter=", ">
+        <text variable="archive"/>
+        <text variable="archive_location"/>
+        <text variable="call-number"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="volume">
+    <choose>
+      <if is-numeric="volume">
+        <text term="volume" form="short" suffix=". "/>
+        <text variable="volume"/>
+        <text variable="number-of-volumes" prefix="/"/>
+      </if>
+      <else>
+        <text variable="volume"/>
+        <text variable="number-of-volumes" suffix=" vol."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="number">
+    <choose>
+      <if is-numeric="number">
+        <text variable="number" prefix="n° "/>
+      </if>
+      <else>
+        <text variable="number"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if is-numeric="issue">
+        <text term="issue" form="short" suffix=" "/>
+        <text variable="issue"/>
+      </if>
+      <else>
+        <text variable="issue"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="interviewer">
+    <group>
+      <names variable="interviewer">
+        <label form="verb" prefix=" " suffix=" "/>
+        <name form="long" and="text" delimiter=", ">
+          <name-part name="family" font-variant="small-caps"/>
+        </name>
+      </names>
+    </group>
+  </macro>
+  <macro name="accessed">
+    <group delimiter=" ">
+      <choose>
+        <if type="graphic personal_communication interview" match="any">
+          <text variable="URL" text-decoration="none" prefix=" URL : " suffix="."/>
+        </if>
+        <else>
+          <text variable="URL" text-decoration="none" prefix=". URL : " suffix="."/>
+        </else>
+      </choose>
+    </group>
+    <group delimiter=" " font-style="normal">
+      <text value="Consulté le" suffix=" " prefix=". "/>
+      <date variable="accessed" form="text">
+        <date-part name="day"/>
+        <date-part name="month"/>
+        <date-part name="year"/>
+      </date>
+    </group>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" prefix=", ">
+      <name et-al-min="3" et-al-use-first="3" initialize-with="."/>
```

Comment:
the 3/3 for et al doesn't make sense. I assume it should be et-al-min="4"? What you have right now is "start using et al with three or more authors and print the first three" That makes no sense for works with exactly three authors.
Same issue above.